src/ray/object_manager/plasma/store.cc

```diff
@@ -64,10 +65,22 @@ ray::ObjectID GetCreateRequestObjectId(const std::vector<uint8_t> &message) {
   return ray::ObjectID::FromBinary(request->object_id()->str());
 }
 
+void ToPlasmaObject(const ObjectTableEntry &entry, PlasmaObject *object,
```

Comment:
Why don't we just return the Plasma object instead of accepting mutable arg?

Reply:
There is some subtlety at the caller of ToPlasmaObject. Let's defer that to follow up PRs.